query/sources/impl/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -12,24 +12,47 @@
  **/
 -->
 <blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
-    <bean id="cswFieldProvider" class="org.codice.ddf.admin.sources.csw.CswFieldProvider">
-        <argument ref="configuratorFactory"/>
-    </bean>
+    <service id="cswFieldProvider" interface="org.codice.ddf.admin.api.FieldProvider">
+        <bean class="org.codice.ddf.admin.sources.csw.CswFieldProvider">
+            <property name="configuratorFactory" ref="configuratorFactory"/>
```

Comment:
These need to be changed to `argument`s to deploy correctly. Same with other beans in this blueprint.